src/pages/notifications/Notifications.jsx

```diff
@@ -70,6 +72,15 @@ export default function Notifications() {
             <List>
               {safeNotifications.map(notification => {
                 const read = get(notification, 'is_read', false);
+                const hasSenderName = Boolean(
```

Comment:
since you use `&&` this is already a boolean

Reply:
Done